doc/en/user/source/data/database/oracle.rst

```diff
@@ -65,7 +65,11 @@ Configuring an Oracle datastore
    * - ``Loose bbox``
      - 	Controls how bounding box filters are made against geometries in the database. See the :ref:`oracle_loose_bbox` section below.
    * - ``Metadata bbox``
-     - 	Flag controlling the use of MDSYS.USER_SDO_GEOM_METADATA or MDSYS.ALL_SDO_GEOM_METADATA table for bounding box calculations, this brings a better performance if the views access is fast and the bounds are configured right in the tables default is false  
+     - 	Flag controlling the use of MDSYS.USER_SDO_GEOM_METADATA or MDSYS.ALL_SDO_GEOM_METADATA table for bounding box calculations, this brings a better performance if the views access is fast and the bounds are configured right in the tables default is false
+   * -  ``Get remarks``
+     -  Boolean flag specifies whether REMARKS metadata will be returned. Note that this parameter is not available in the JNDI version of the DataStore. Instead you can set a Connection Property called remarksReporting as described in the Oracle `documentation`_.
+
+.. _documentation: https://docs.oracle.com/cd/A97335_02/apps.102/a83724/basic1.htm#1024018
```

Comment:
This is the Oracle 8 doc... since we are now at 19+
```suggestion
.. _documentation: 
https://docs.oracle.com/en/database/oracle/oracle-database/19/jjdbc/performance-extensions.html#GUID-FEECA64F-44F4-453F-B8A8-AFBF6D29ABA4
```

Reply:
Thanks.  Updated and pushed with a slightly more specific link from the same doc.